stf/model/capacity_prognosis_model.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, classes=None, hyper_params=None):
         # definition of classification classes
         self.classes = classes
 
-        # create xgb model
+        # create lgb model
         self.model = xgb.XGBClassifier(kwargs=self.hyper_params)
```

Comment:
I think you still want to put 'xgb' here as a XGBClassifier is still used in this case

Reply:
I'll change it, thanks!